libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/TARGET_NRF51822_MKIT/PinNames.h

```diff
@@ -61,7 +61,7 @@ typedef enum {
     p28 = 28,
     p29 = 29,
     p30 = 30,
-//    p31=31,    
+    p31 = 31,    
```

Comment:
The [official documentation](https://www.nordicsemi.com/eng/nordic/download_resource/30462/1/50872152) specifies 31 GPIOs. When I use it I need to import [mbed-src](https://developer.mbed.org/users/mbed_official/code/mbed-src/) and modify the line manually, but that is not the proper solution.